### DISCUSSION TITLE
From a pure MQTT pov, how to detect motion?

### DISCUSSION THREAD

**Discussion by dmartinpro** [blakeblackshear/frigate#6177] [Ask A Question]
Hi,

What is the best and effective way to detect motion from MQTT information?
Which elements should be taken into account?
My objective is to trigger an Openhab rule to send a msg.

**Comment by NickM-27** [comment]
Probably the motion topic https://docs.frigate.video/integrations/mqtt#frigatecamera_namemotion

**Reply by dmartinpro** [reply]
You're right, this is the most simple way. My need was slightly different and is more related to detecting an event (a "validated" motion is a way) as Motion seems to be ON too often in my case.
My need is more about triggering something when a "real/validated" motion is detected (meaning the root cause of the motion is identified and is not a false positive). Of course, processing the event payload is possible, but a simple boolean could have been more convenient.
Is such a boolean property an option which could be considered in the future as a dedicted MQTT node?

**Reply by NickM-27** [reply]
I think it would be better if we use correct terms. Motion is motion, and an object is an object. You can use the 'all' topics for count if all objects and there's also a Boolean for checking if any object is currently detected.
